src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -0,0 +1,194 @@
+                <ul className="stem-ad-list-secondary">
+                  <li className="stem-ad-list-item">
+                    "Are you enrolled in a science, technology, engineering, or
+                    math (STEM) degree program?" or
+                  </li>
+                  <li className="stem-ad-list-item">
+                    "Do you have a STEM undergraduate degree and are now
+                    pursuing a teaching certification?"
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+        )}
+      </ul>
+      <h3 className="vads-u-font-family--sans">
+        What you should do if you disagree with our decision
+      </h3>
+      <p>
+        If you disagree with our decision, you have 1 year from the date of this
+        letter to request a decision review or appeal. You have 3 options to do
+        this:
+      </p>
+      <ul className="stem-ad-list">
+        <li className="stem-ad-list-item">
+          If you have new or previously unprovided evidence to prove your
+          eligibility for this scholarship, you can file a Supplemental Claim by
+          completing VA Form 20-0995.{' '}
+          <a href="https://www.va.gov/find-forms/about-form-20-0995/">
+            Download VA Form 20-0995.
+          </a>
+        </li>
+        <li className="stem-ad-list-item">
+          If you don't have new evidence but would like a more senior reviewer
+          to look at your case, you can request a Higher-Level Review by
+          completing VA Form 20-0996.{' '}
+          <a href="https://www.va.gov/find-forms/about-form-20-0996/">
+            Download VA Form 20-0996.
+          </a>
+        </li>
+        <li className="stem-ad-list-item">
+          If you don't agree with the decision on your Supplemental Claim or
+          Higher-Level Review, you can appeal to a Veterans Law Judge by
+          completing VA Form 10182.{' '}
+          <a href="https://www.va.gov/find-forms/about-form-10182/">
+            Download VA Form 10182.
+          </a>
+        </li>
+      </ul>
+      <p>
+        You can read more about these options on VA Form 20-0998, "Your Rights
+        to Seek Further Review of Our Decisions".{' '}
+        <a href="https://www.va.gov/find-forms/about-form-20-0998/">
+          Download VA Form 20-0998
+        </a>
+        . You can also contact us at (888) GI BILL 1 (888-442-4551) to request
+        any of these forms.
+      </p>
+      <p>
+        <a href="https://www.va.gov/decision-reviews">
+          Learn more about the decision review process.
+        </a>{' '}
+        If you would like to review the evidence we used in our decision, please
+        contact us. You may be able to review some evidence by signing in to
+        your account on <a href="https://www.va.gov/">https://www.va.gov</a>.
+      </p>
+      <h3 className="vads-u-font-family--sans">More resources</h3>
+      <table>
```

Comment:
Curious about the decision to go with a table for these links. It seems like it'd be more concise to list them, instead of having separate `th` tags and `td` tags for what appears to be link text and href values.

Reply:
All of the table related items you mentioned were also part of the design prototype we had chosen to go with. This includes the usages of the table as well as the choice to use the URL's as link text within the table itself.

Reply:
Have you tested what the table looks like for mobile devices? Also, have y'all worked with the platform's a11y specialists regarding the use of URLs as link text? Only asking because it feels as though there are some accessibility concerns there, and would hate for y'all to be roadblocked closer to launch on something like that. These URLs are not screen reader friendly.